<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QArrangementInfoWidget</class>
 <widget class="QWidget" name="QArrangementInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>194</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arrangement info</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/ArrangementDemoWindow.qrc">
    <normaloff>:/dialog-information.png</normaloff>:/dialog-information.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="_labelFaces">
     <property name="text">
      <string>0 faces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelEdges">
     <property name="text">
      <string>0 edges</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelVertices">
     <property name="text">
      <string>0 vertices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelIsolatedVertices">
     <property name="text">
      <string>0 isolated vertices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelVerticesAtInfinity">
     <property name="text">
      <string>0 vertices at infinity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_labelChanged">
     <property name="text">
      <string>No new changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/ArrangementDemoWindow.qrc"/>
 </resources>
 <connections/>
</ui>
